lib/galaxy/managers/configuration.py

```diff
@@ -73,7 +73,8 @@ def _defaults_to(default):
             # TODO: is there no 'correct' way to get an api url? controller='api', action='tools' is a hack
             # at any rate: the following works with path_prefix but is still brittle
             # TODO: change this to (more generic) upload_path and incorporate config.nginx_upload_path into building it
-            'nginx_upload_path'                 : lambda i, k, **c: getattr(i, k, False) or self.url_for('/api/tools'),
+            'nginx_upload_path'                 : lambda i, k, **c: getattr(i, k, False),
+            'chunk_upload_size'                 : _defaults_to(0),
```

Comment:
Should this default to `100` as above?

Reply:
I am not sure but I changed this. Previously I wanted to avoid redundancies since the default has already been set at the config level. Thanks for looking into it.